package main.java.com.revature.exceptions;

public class UserNotFoundException extends RuntimeException
{

}
